src/main/webapp/pages/ou.html

```diff
@@ -89,56 +89,111 @@ <h2>Organizational Units</h2>
-
-                        function setTransform() {
-                        treeChart.style.transform = "translate(" + xoff + "px, " + yoff + "px) scale(" + scale + ")";
-                        }
-
-                        treeChart.onmousedown = function(e) {
-                        e.preventDefault();
-                        start = {x: e.clientX - xoff, y: e.clientY - yoff};    
-                        panning = true;
-                        }
-
-                        treeChart.onmouseup = function(e) {
-                        panning = false;
-                        }
-
-                        treeChart.onmousemove = function(e) {
-                        e.preventDefault();         
-                        if (!panning) {
-                            return;
-                        }
-                        xoff = (e.clientX - start.x);
-                        yoff = (e.clientY - start.y);
-                        setTransform();
-                        }
-
-                        treeChart.onwheel = function(e) {
-                            e.preventDefault();
-                            // take the scale into account with the offset
-                            var xs = (e.clientX - xoff) / scale,
-                                ys = (e.clientY - yoff) / scale,
-                                delta = (e.wheelDelta ? e.wheelDelta : -e.deltaY);
-
-                            // get scroll direction & set zoom level
-                            (delta > 0) ? (scale *= 1.2) : (scale /= 1.2);
-
-                            // reverse the offset amount with the new scale
-                            xoff = e.clientX - xs * scale;
-                            yoff = e.clientY - ys * scale;
-
-                            setTransform();          
-                        }
-                    </script>
+                </div>
+            </div>
+        </div>
+        <!-- Outside of flex container -->
+
+        <div class="container">
+            <div class="console">
+                <div class="console-banner">
+                    <div class="console-title">Edit OUs</div>
+                </div>
+                <div class="console-content">
+                    <div class="subheader">Select an Action:</div>
+                    <div class="form-group">
+                        <label for="edit-choice">You can either create a new OU, update an existing OU, or delete an existing OU.</label>
+
+                        <select class="form-control" id="edit-choice">
+                            <option value="create">Create</option>
+                            <option value="update">Update</option>
+                            <option value="delete">Delete</option>
+                        </select>
+                    </div>
+
+                    <!-- Deletion -->
+                    <form style="display: none;" id="edit-delete" onSubmit="deleteOU();return false">
```

Comment:
This console is fine for now, but if possible in later iterations see if it is possible to move some of this code to another file so that ou.html does not get too large